integration-cli/docker_cli_import_test.go

```diff
@@ -39,3 +42,59 @@ func (s *DockerSuite) TestImportDisplay(c *check.C) {
 	}
 
 }
+
+func (s *DockerSuite) TestImportFile(c *check.C) {
+	runCmd := exec.Command(dockerBinary, "run", "--name", "test-import", "busybox", "true")
+	out, _, err := runCommandWithOutput(runCmd)
+	if err != nil {
+		c.Fatal("failed to create a container", out, err)
+
+	}
+
+	temporaryFile, err := ioutil.TempFile("", "exportImportTest")
+	if err != nil {
+		c.Fatal("failed to create temporary file", "", err)
+	}
+
+	runCmd = exec.Command(dockerBinary, "export", "test-import")
+	runCmd.Stdout = bufio.NewWriter(temporaryFile)
+
+	_, err = runCommand(runCmd)
+	defer os.Remove(temporaryFile.Name())
+	if err != nil {
+		c.Fatal("failed to export a container", out, err)
+	}
+
+	runCmd = exec.Command(dockerBinary, "import", temporaryFile.Name())
+	out, _, err = runCommandWithOutput(runCmd)
+	if err != nil {
+		c.Fatal("failed to import a container", out, err)
+	}
+
+	if n := strings.Count(out, "\n"); n != 1 {
+		c.Fatalf("display is messed up: %d '\\n' instead of 1:\n%s", n, out)
+	}
+	image := strings.TrimSpace(out)
+
+	runCmd = exec.Command(dockerBinary, "run", "--rm", image, "true")
+	out, _, err = runCommandWithOutput(runCmd)
+	if err != nil {
+		c.Fatal("failed to create a container", out, err)
+	}
+
+	if out != "" {
+		c.Fatalf("command output should've been nothing, was %q", out)
+	}
+
+}
+
+func (s *DockerSuite) TestImportFileNonExistentFile(c *check.C) {
+
+	runCmd := exec.Command(dockerBinary, "import", "non-existent-file")
```

Comment:
Can you do me a favor?  In this test change `non-existent-file` to `example.com/myImage.tar`. This should still fail the same way but I want to make sure that we **never** interpret this string as a URL.  Prior to this PR all strings would be passed to the daemon as a URL and it would try to download from it.  So this PR does change this behavior, but I'm in favor of it.  However, I think we need to add a test for it (by using a string that almost looks like a url, but isnt').
Also, add a comment to mentioning all of this so that people know why you picked such a url-ish filename.